src/style.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins');
```

Comment:
- [ ] I would suggest adding some spacing to the sides because currently your design is aligned to the left and according to the wireframe the design should be centered. Please, add some space to the sides or center the elements of your app :pushpin: :smiley:

| Wireframe | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/147315130-39dae4b6-c2b8-4ca4-b26a-9745f9e3c652.png) | ![image](https://user-images.githubusercontent.com/81522435/147315096-daa5d316-8bf7-4a0f-8795-33ce8376f6fc.png) |